packages/firestore/test/unit/remote/node/serializer.test.ts

```diff
@@ -337,220 +346,10 @@ describe('Serializer', () => {
           }
         }
       };
-      const obj = s.toValue(objValue);
-      expect(obj).to.deep.equal(expectedJson);
-
-      // Compare the raw object representation rather than the Message because
-      // occasionally Jasmine will hang trying to generate the string form in
-      // the event of an inequality.
-      expect(new ds.Value(obj).toRaw(true, true)).to.deep.equal(
-        new ds.Value(expectedJson).toRaw(true, true)
-      );
-      // clang-format on
-    });
-  });
+      const proto: api.Value = expectedJson;
```

Comment:
Regarding this comment which I deleted: "Compare the raw object ... in the event of an inequality".

After my change (I didn't test it before my change), I found this to be incorrect. If I introduced an inequality and ran the tests a few times, the tests always completed with the error string properly generated. As a result, I eliminated the special casing here.